Feature/generalize module (#3)
qbeyond/terraform-azurerm-vm-monitoring: README.md

# ALZ Monitoring
[![GitHub tag](https://img.shields.io/github/v/tag/qbeyond/terraform-azurerm-vm-monitoring.svg)](https://registry.terraform.io/modules/qbeyond/vm-monitoring/azurerm/latest)

> [![License](https://img.shields.io/github/license/qbeyond/terraform-azurerm-vm-monitoring.svg)](https://github.com/qbeyond/terraform-azurerm-vm-monitoring/blob/main/LICENSE)

Reviewer: Change according to new planned name

> This is a module for creating monitoring rules on top of a central log analytics workspace. the query rules and output are compatible with the event pipeline for central monitoring of q.beyond AG.

Reviewer 2: ```suggestion
This is a module for creating monitoring rules on top of a central log analytics workspace. The query rules and output are compatible with the event pipeline for central monitoring of the q.beyond AG.
```

> This is a module for creating monitoring rules on top of a central log analytics workspace. the query rules and output are compatible with the event pipeline for central monitoring of q.beyond AG.

Reviewer: ```suggestion
This module deploys all resources to enable the monitoring of a Log Analytics Workspace for all managed resources by the managed services provider. This includes Action group to send alerts to event pipeline of MSP, alerts for specific resources and a automation to add information (especially tags) to the LAW.
```

> It also creates a resourcegraph with an automation account.

Reviewer 2: ```suggestion
To enable usage of tags and resource properties in monitoring queries an existing automation account is used to import resource metadata into the central log analytics workspace.
```

> *This module will not work if LAW with same name is in soft delete state.* You can use a new name or [force delete](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/delete-workspace#permanent-workspace-delete). 

Reviewer: ```suggestion
```
As this module does not create a LAW, i think this is not needed.
